### PR TITLE
ADCM-1807 Add pre-filled ADCM to func tests

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 --extra-index-url https://ci.arenadata.io/artifactory/api/pypi/python-packages/simple
 adcm-client>=2021.6.21.15
-adcm_pytest_plugin>=4.2.4
+adcm_pytest_plugin==4.4.0
```
Comment (vgorkavenko) [inline requirements-test.txt:3]: Are you sure that not `>=` ?

**Review by skhomuti — APPROVED** [review]

File: requirements-test.txt (hunk 1 above)
Comment (madlabman) [inline requirements-test.txt:3]: `==`?

**Review by madlabman — APPROVED** [review]

File: requirements-test.txt (hunk 1 above)
Comment (Sealwing) [inline requirements-test.txt:3]: Maybe fix minor like >=4.4.0, <4.5.0
Or there is a reason for strict version fix?

**Review by Sealwing — APPROVED** [review]

File: requirements-test.txt (hunk 1 above)
Comment (dgusakov) [reply requirements-test.txt:3]: Thx forgot to change

**Review by Sealwing — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Review by madlabman — APPROVED** [review]

**Pull request merged**